### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by Noordfrees**
Update changelog up to 6c9962ab5307f4271126531d46e0414770ec7311

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -83,6 +171,33 @@
...
+- Replace some asserts in the AI with more informative Exceptions for tribe/world designers. (#3524)
+- Limit ware/worker preciousness for the AI. (#3530): Warn if ware/worker preciousness is a bit high and throw an exception if it's over 200.
+- AI supports seafaring (#3528). This introduces new ai_hint "supports_seafaring", that will be used to boost building score for particular building when on seafaring map.
+- AI fixes (#3602)
+  - fixed crude percent to prevent overflow
+  - AI worker management
+  - implemented a worker management for AI Productionsites and mines.
+  - added a malus for unoccupied sites to the build decisions.
+  - some small improvements for upgradehandling of buildings
+- About 100 rounds of training
```
Comment (hessenfarmer) [inline ChangeLog]: all training rounds should be handled by one line I wouln't even write down the number of rounds. 
Proposal: Use "further training of AI as placeholder

File: ChangeLog [hunk 2]
```diff
@@ -83,6 +171,33 @@
...
+- Results of about 100 rounds of training
+- Used crude percent from AI for performance statistics strineg in productionsites
```
Comment (hessenfarmer) [inline ChangeLog]: The line above is not an AI fix but an UI change

File: ChangeLog [hunk 3]
```diff
@@ -113,6 +228,70 @@
   "built_over_immovable" = <immovable attribute>
 - Fixed bug #1642882: Port should clear enough land to build
 - Fixed bug #1827033: Fix an assert fail because of nullptr destinations in the shipping algorithm
+- Change Barbarian evade increase per level from 15 to 16.
+- Make Empire Piggery work 20% faster
+- Empire Vineyards plant a bit faster
+- Made piggeries work as fast as donkeybreeders
```
Comment (hessenfarmer) [inline ChangeLog]: The above 4 lines could be moved to balancing tweaks

File: ChangeLog [hunk 4]
```diff
@@ -208,6 +564,99 @@
...
+- Fix packaging for Focal and #3783
+- Fix travis MacOS builds (#3848): On Travis MacOS, gettext needs to be added to the PATH and some CMake CXX and linker flags have to be set because brew force-linking is no longer allowed for whatever reason
+- Fix travis failure on gcc-9.1_debug (#3861)
+- Improve SDL2 detection on modern systems (#3872)
+- Add Git hash to AppVeyor and bzr version numbers (#3894)
+- updated innosetup file to reflect new Icu filenames (#3903)
+- gcc-10 compatibility (#3908)
+- Replaced all uses of boost::bind with lambda functions and added a codecheck rule to deprecate boost::bind() and boost::ref()
+- Added missing DLLs to appveyor builds (#3917)
+- Appveyor cleanup (#3919)
```
Comment (hessenfarmer) [inline ChangeLog]: I would remove the above 2 entries as they are not related top widelands but to MSYS2 changes. If you want to have them you could write various fixes to the appveyor environment

**Comment by hessenfarmer** [comment]
ok from my side. maybe better to wait with merging untiol the last thing is in.

**Review by hessenfarmer — APPROVED** [review]

**Comment by klaus-halfmann** [comment]
Why could this not be merged, yet?

As you may have seen I am (mostly) back and can offer a bit of spare time.
Please hint me what woth reviewing next.

**Comment by Noordfrees** [comment]
This is already approved but I'll only merge it a few days before the winter time freeze so that nothing will be left out.

**Comment by Noordfrees** [comment]
> As you may have seen I am (mostly) back and can offer a bit of spare time.
> Please hint me what woth reviewing next.

Great :) 
Now the only thing left to review is #3850 and as soon as that is also merged I will schedule the winter time freeze and announce the dates for the release :)

**Pull request merged**